src/index.js

```diff
@@ -1,8 +1,21 @@
 const {ApolloServer, gql} = require('apollo-server')
+const { Sequelize } = require('sequelize');
+const sequelize = new Sequelize('postgres://localhost:5432/barrettebanner')
```

Comment:
this opens up the need for dotenv in node. Don't worry it's way easier setting it up here than in angular. You don't want to expose ur db connection url in ur public repo